doc/rolling_update.md

```diff
@@ -0,0 +1,283 @@
+# Rolling update
+
+Here is described rolling update for an application that uses
+TGZ package with [multiversion approach](/doc/multiversion.md) or
+usual RPM/DEB packages.
+
+## Requirements
+
+* `tarantool.cartridge` >= `1.9.0`
+
+## Leaders promotion
+
+[`failover_promote`](/doc/scenario.md#failover_promote) step can be used
+for leaders promotion only if stateful failover is enabled.
+
+To specify leaders that should be promoted `failover_promote_params` variable
+should be used. It's a dictionary with fields:
+
+- `replicaset_leaders` (`dict`): describes the mapping between replica sets
+and leaders aliases;
+- `force_inconsistency` (`bool`): make promotion forcefully, don't wait for the
+consistent switchover.
+
+## Rolling update: Plan
+
+Imagine that you have a cluster with `myapp-1.0.0` deployed.
+And now you want to update your application to `myapp-2.0.0`.
```

Comment:
Maybe it's not good to start sentence with "and":
https://community.languagetool.org/rule/show/And?lang=en&subId=1